upstairs/src/downstairs.rs

```diff
@@ -4521,8 +4517,11 @@ pub(crate) mod test {
                 None,
             );
         }
-
-        ds.ack(ds_id);
+        // Writes are fast-acked when first submitted
+        if !matches!(ds.ds_active.get(&ds_id).unwrap().work, IOop::Write { .. })
```

Comment:
Even though this is just for tests, should this check for WriteUnwritten here too?

Reply:
Yeah, done in 28f79c258115